### ISSUE TITLE
Error: wb_supervisor_field_get_count() can only be used with multiple fields (MF).

### ISSUE THREAD

**Issue by Justin-Fisher** [cyberbotics/webots#4356]
Webots Supervisor docs say:  
> The wb_supervisor_field_get_count function returns the number of items of a multiple field (MF) passed as an argument to this function. If a single field (SF) or NULL is passed as an argument to this function, it returns -1. Hence, this function can also be used to test if a field is MF (like WB_MF_INT32) or SF (like WB_SF_BOOL).

If you do call `wb_supervisor_field_get_count()` on an SF field, it gives the expected value -1, but also prints a warning: "Error: wb_supervisor_field_get_count() can only be used with multiple fields (MF)."

According to the documentation, returning -1 is the intended useful behavior here, so there should be no need for the warning.  Or, if we aren't supposed to use this on SF fields, then the documentation shouldn't suggest this as legitimate usage.

**Comment by omichel** [comment]
I agree, we should either get rid of this warning or remove the fact that this function can be used to test the MF/SF type of a field.

**Comment by omichel** [comment]
My preference would go for the removal of the warning, as we don't have a simple way to determine if we have a SF or a MF field type.

**Comment by Justin-Fisher** [comment]
My somewhat simple way of testing SF-vs-MF has been checking if 'SF' is a substring of the field's typename, but if this could do it faster, that'd be good.
